controllers/nginx/configuration.md

```diff
@@ -340,22 +342,23 @@ The next table shows the options, the default value and a description
 |use-gzip|"true"|
 |use-http2|"true"|
 |vts-status-zone-size|10m|
-|worker-processes|<number of CPUs>|
+|worker-processes|number of CPUs|
```

Comment:
maybe something like "$(nproc)" would be clearer, since it isn't literally being set to 'number of CPUs'.

I'm not sure the best option.